### PR TITLE
fix: errors and warnings in frontend

### LINKED ISSUE #210: Duplicate key cause table errors
https://user-images.githubusercontent.com/24910914/127731784-a619ee85-2695-41a0-8347-58ac379e42a9.mp4

**All key-based components may occur this problem.**

This error cause we use the wrong or duplicate `rowKey` to build a table, every time we click the `Add` button, it generates a row like

```json
{
    "key": "Please select a signup item",
    "value": "..."
}
```

I fixed part of the errors in `web/src/UrlTable.js`: [`860fe91`](https://github.com/casbin/casdoor/commit/860fe91725865f202c2e282fe956647a6d4674f8#diff-aa513e39552689c307c054b430526e17ef206f1db173d3cb4c3705ee20d3d1abL98-R98)

### PR DESCRIPTION
**Pull request by WindSpiritSR**
1. Fix #210 
2. Add `key` for form item, like `<React.Fragment key="emailBox">`, these changes can fix some warning in dev mode, no effect to production mode
3. Support case insensitivity search for `SelectRegionBox`
4. Replace single quotes with double quotes
5. Reformat

Signed-off-by: WindSpiritSR <simon343riley@gmail.com>

### CODE REVIEW

**Comment by hsluoyz** [comment]
@Kininaru @oranges-eating @turbodog03 plz review

File: web/src/App.js [hunk 1]
```diff
@@ -367,30 +366,31 @@ class App extends Component {
       window.location.pathname === '/';
   }
 
-  renderRouter(){
-    return(
+  renderRouter() {
+    //TODO: Check account is null and is signined
```
Comment (turbodog03) [inline web/src/App.js:370]: I wonder if this necessary? Bause the following functions have the same effect.
```
this.renderLoginIfNotLoggedIn(<HomePage account={this.state.account} {...props} />)
```

**Review by turbodog03 — COMMENTED** [review]
```
//TODO: Check account is null and is signined
```
I wonder if this necessary? Bause the following functions have the same effect.
```
this.renderLoginIfNotLoggedIn(<HomePage account={this.state.account} {...props} />)
```

**Comment by WindSpiritSR** [comment]
@turbodog03 Forgotten, already remove

**Review by oranges-eating — APPROVED** [review]
lgtm

**Comment by hsluoyz** [comment]
@WindSpiritSR fix:

image

**Pull request closed without merging**